cirq/google/op_serializer.py

```diff
@@ -200,6 +201,14 @@ def _arg_value_to_proto(self, value: arg_func_langs.ArgValue,
             msg.arg_value.bool_values.values.extend(value)
         elif isinstance(value, sympy.Symbol):
             msg.symbol = str(value.free_symbols.pop())
+        elif isinstance(value, sympy.Add):
```

Comment:
You need to also set the arg func language in program.proto (don't forget there are current two copies during migration of paths)
`arg_function_language = 2;`

Reply:
Urgh it's so much worse than that. Nothing is passing around the language at the moment, or even setting it to something. I have to plumb it the whole way through.